README.md

```diff
@@ -10,6 +10,53 @@ official packaging.
 
 ## Usage
 
+### Documentation
```

Comment:
I did feel a little weird about calling it documentation for just that reason. I was trying to convey the fact that this section contains technical documentation of the function contracts.